src/components/Earn.jsx

```diff
@@ -327,8 +326,6 @@ export default function Earn() {
     } catch (e) {
       setServiceInfoAlert(null)
       setAlert({ variant: 'danger', message: e.message })
-    } finally {
-      setSubmitting(false)
```

Comment:
We're these calls not doing anything before?

Reply:
Formik resets the flag automatically when using `async` functions.